scripts/performance/perf_load/perf_gen_req_parser.py

```diff
@@ -23,7 +25,7 @@ class ReqTypeParser:
          "get_revoc_reg_delta": RGGetRevocRegDelta, "get_payment_sources": RGGetPaymentSources, "payment": RGPayment,
          "verify_payment": RGVerifyPayment, "cfg_writes": RGConfigChangeState, "demoted_node": RGPoolNewDemotedNode,
          "get_txn": RGGetTxn, 'get_taa': RGGetTAA, 'get_taa_aml': RGGetTAAAML, 'get_auth_rule': RGGetAuthRules,
-         "get_fees": RGGetFees}
+         "get_fees": RGGetFees, "demote": RGDemoteNode, "promote": RGPromoteNode}
```

Comment:
Please update Readme with the info about new requests. Please mention there how to use them (that rate needs to equal, and promotion shift must be set)

Reply:
Done.